integration-tests/google-storage/src/main/java/org/apache/camel/quarkus/component/google/storage/it/RealAccountPropertyProducer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.google.storage.it;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+import org.eclipse.microprofile.config.spi.ConfigSourceProvider;
+
+public class RealAccountPropertyProducer implements ConfigSourceProvider {
+
```

Comment:
Can't we simply use env var from properties file, something like https://quarkus.io/guides/config-reference#combine-property-env-var ?

Reply:
@aldettinger this is a nice idea. I already tried it and I was facing an issue with it. But now I think that it should be possible to figure it out. I'll try it once again. Thanks for suggestion!

Reply:
@aldettinger it works now, thanks again.